src/test/java/com/wikia/webdriver/testcases/desktop/createawikitests/CreateWikiTestsLang.java

```diff
@@ -25,8 +25,8 @@ public void createNewWikiLangTC001(String lang) {
     WikiBasePageObject base = new WikiBasePageObject();
     CreateNewWikiPageObjectStep1 cnw1 = base.openSpecialCreateNewWikiPage(wikiCorporateURL);
     cnw1.selectLanguage(lang);
-    String expectedDomainPrefix = "en".equals(lang) ? "" : String.format("%s.", lang);
-    assertEquals(cnw1.getDomainPrefix().getText(), expectedDomainPrefix);
+    String expectedDomainSufix = "en".equals(lang) ? "" : String.format("%s", lang);;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this empty statement. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AEmptyStatementUsageCheck)